### PR TITLE
MES-1035 - Journal rework

### PR DESCRIPTION
**Pull request by johnmorleybjss**
Rework the Journal page as per the new designs.

- PO signoff [x]
- QA signoff [x]

img_b6a3509992ae-1

### CODE REVIEW

File: src/components/journal-header/journal-header.scss [hunk 1]
```diff
@@ -1,74 +1,29 @@
...
+    text-decoration: underline;
+    font-size: 16px;
+  }
+  span.journal-header-today-title {
+    font-size: 48px;
+    font-weight: bold;
+  }
+  span.journal-header-date {
+    font-size: 24px;
+    font-weight: bold;
```
Comment (Cavalier07) [inline src/components/journal-header/journal-header.scss]: Could you not use mes-header-lg instead?

Comment (johnmorleybjss) [reply src/components/journal-header/journal-header.scss]: Yes, I will refactor this now. Perhaps I could also migrate `span.journal-header-today-title` into `_typography.scss` as `mes-title` - thoughts?

Comment (johnmorleybjss) [reply src/components/journal-header/journal-header.scss]: Can't see another instance of a title like on the journal, will leave that one as it is.

File: src/assets/imgs/warning.svg [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ammarhaiderbjss) [inline src/assets/imgs/warning.svg]: where'd this file come from?

Comment (johnmorleybjss) [reply src/assets/imgs/warning.svg]: Wikimedia, does it need some sort of sign-off?

File: src/components/journal-candidate-info/journal-candidate-info.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
+journal-candidate-info {
+  & > div {
+    height: 100%;
+    padding: 0.3em 0.75em;
+    background-color: map-get($colors, gds-grey-4);
+    border-radius: 13px;
+    -webkit-box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.55);
```
Comment (ammarhaiderbjss) [inline src/components/journal-candidate-info/journal-candidate-info.scss]: could create a sass `mixin` for this instead. Something like this perhaps?
```
// Mixin implementation
@mixin box-shadow($top, $left, $blur, $color) {
  -webkit-box-shadow: $top $left $blur $color;
  -moz-box-shadow: $top $left $blur $color;
  box-shadow: $top $left $blur $color;
}
// Usage..
@include box-shadow(0px, 2px, 4px, rgba(0, 0, 0, 0.55));
```

File: src/components/journal-candidate-info/journal-candidate-info.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    -moz-box-shadow: 0px 2px34px 0px rgba(0, 0, 0, 0.55);
```
Comment (ammarhaiderbjss) [inline src/components/journal-candidate-info/journal-candidate-info.scss]: think this is typo

File: src/pages/post-test-summary/post-test-summary.ts [hunk 5]
```diff
@@ -125,7 +125,7 @@ export class PostTestSummaryPage {
   private backToJournal() {
     this.faultStore.reset();
     this.summaryMetadata.reset();
-    this.navCtrl.popToRoot();
+    this.navCtrl.popTo(this.navCtrl.getByIndex(1));
```
Comment (ammarhaiderbjss) [inline src/pages/post-test-summary/post-test-summary.ts]: any specific reason for changing this?

Comment (johnmorleybjss) [reply src/pages/post-test-summary/post-test-summary.ts]: In order to show the back button on the Journal page, I had to remove some code which pruned the Welcome page out of the navigation stack. This meant that `popToRoot` was sending us to the welcome page after test completion.

File: package.json [hunk 6]
```diff
@@ -66,6 +66,7 @@
     "ionic-angular": "3.9.2",
     "ionicons": "3.0.0",
     "lodash": "^4.17.5",
+    "moment": "^2.22.2",
```
Comment (ammarhaiderbjss) [inline package.json]: thank you

File: src/assets/imgs/warning.svg (hunk 2 above)
Comment (ammarhaiderbjss) [reply src/assets/imgs/warning.svg]: As long as UX are aware it's fine

Comment (johnmorleybjss) [reply src/assets/imgs/warning.svg]: I will check with them for confirmation.

File: src/components/journal-candidate-info/journal-candidate-info.scss (hunk 3 above)
Comment (johnmorleybjss) [reply src/components/journal-candidate-info/journal-candidate-info.scss]: Done

File: src/components/journal-candidate-info/journal-candidate-info.scss (hunk 4 above)
Comment (johnmorleybjss) [reply src/components/journal-candidate-info/journal-candidate-info.scss]: Fixed with mixin.

File: src/assets/imgs/warning.svg (hunk 2 above)
Comment (johnmorleybjss) [reply src/assets/imgs/warning.svg]: Replaced the Wikimedia SVG with a PNG provided by UX.

**Comment by ammarhaiderbjss** [comment]
sorry, last one, can we remove changes to `package-lock.json`?

**Pull request merged**